package.json

```diff
@@ -21,17 +21,22 @@
   },
   "homepage": "https://github.com/teddy4445/lecture_website_app_generator#readme",
   "dependencies": {
+    "@hapi/joi": "^17.1.1",
     "axios": "^0.19.2",
+    "bcryptjs": "^2.4.3",
+    "bootstrap": "^4.5.2",
```

Comment:
I recall @arielfenster said we won't use bootstrap in this project...

Reply:
fixed